fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java

```diff
@@ -602,7 +602,7 @@ public static TextFileFormatDesc toTextFileFormatDesc(Map<String, String> parame
         if (fieldDelim.isEmpty()) {
             // Support for hive org.apache.hadoop.hive.serde2.OpenCSVSerde
             // https://cwiki.apache.org/confluence/display/hive/csv+serde
-            fieldDelim = parameters.getOrDefault(OpenCSVSerde.SEPARATORCHAR, "");
+            fieldDelim = parameters.getOrDefault(OpenCSVSerde.SEPARATORCHAR, ",");
```

Comment:
1. maybe you can use `CSVWriter.DEFAULT_SEPARATOR` instead of `,`
2. could you help to make it clear how it works if there is an array column?

Reply:
> 1. maybe you can use `CSVWriter.DEFAULT_SEPARATOR` instead of `,`
> 2. could you help to make it clear how it works if there is an array column?

1. updated
2. csv datasource dose not support array type

Reply:
@zombee0 You are right . array column will return wrong data, I updated to dedicate the Serde lib